framework/src/mesh/MooseMesh.C

```diff
@@ -1299,38 +1310,41 @@ MooseMesh::detectPairedSidesets()
 
         // x-direction faces
         if (normals[0].absolute_fuzzy_equals(minus_x))
-          minus_x_ids.insert(face_ids.begin(), face_ids.end());
+          minus_x_ids[i].insert(face_ids.begin(), face_ids.end());
```

Comment:
What does this do if there are no ids?  Can we short-circuit this whole logic by checking that first?